### ISSUE TITLE
Feature request: blur and crt filter for N64 emulator

### ISSUE THREAD

**Issue by MrMabulous** [rileytestut/Delta#181]
When playing N64 games on a CRT TV, one couldn't see any aliasing because the hardware added a blur to the output and the CRT TV further changed the signal.

To recreate the original feel of playing N64 games, this blur plus the effects of a CRT TV should be added, which should be simple to implement as a full-screen fragment shader.

**Comment by dansl** [comment]
Adding shaders and filters in general for all cores would be amazing. Love playing GBC and GBA with an LCD filter in RetroArch.

**Comment by albatrosify** [comment]
is there any news on this? or anyone planning to implement this. I've tried this on retroArch and absolutely love it.
But I love the UI of Delta much more (and the skins) and therefore will not switch to retroArch...

**Comment by danysgit** [comment]
Please add this feature

**Comment by KingMob** [comment]
Yes, please add it. The original artists back in the day designed sprites to be seen on CRTs.

Great app otherwise!

**Comment by stunrelay** [comment]
+1 for shader options per core (e.g lcd on handhelds, CRT on home consoles)

**Comment by dansl** [comment]
I started using Manic Emu app, since it has shaders.
